test/spec/api/v4/dataview/category.spec.js

```diff
@@ -0,0 +1,290 @@
+var Backbone = require('backbone');
+var _ = require('underscore');
+var carto = require('../../../../../src/api/v4/index');
+
+function createInternalModelMock () {
+  var internalModelMock = {
+    set: function () {},
+    get: function () {}
+  };
+  spyOn(internalModelMock, 'set');
+  spyOn(internalModelMock, 'get').and.callFake(function (key) {
+    switch (key) {
+      case 'count': return 42;
+      case 'max': return 9;
+      case 'min': return 1;
+      case 'nulls': return 0;
+      case 'data': return [
+        {
+          name: 'cat1',
+          value: 1,
+          agg: false
+        },
+        {
+          name: 'others',
+          value: 9,
+          agg: true
+        }
+      ];
+    }
+  });
+  _.extend(internalModelMock, Backbone.Events);
+
+  return internalModelMock;
+}
+
+function createSourceMock () {
+  return new carto.source.Dataset();
+}
+
+function createEngineMock () {
+  var engine = {
+    name: 'Engine mock',
+    reload: function () {}
+  };
+  spyOn(engine, 'reload');
+
+  return engine;
+}
+
+describe('category dataview public v4 API', function () {
```

Comment:
We normally use the file path here. eg: `describe('src/api/v4/dataview/category', ...)`.

Reply:
(it makes filtering specs easier)

Reply:
Ok

Reply:
Ooops! I did that also 😞